src/main/java/org/vafer/jdeb/maven/DebMojo.java

```diff
@@ -359,6 +360,16 @@
     @Parameter(defaultValue = "gnu")
     private String tarBigNumberMode;
 
+    /**
+     * Timestamp for reproducible output archive entries, either formatted as ISO 8601
+     * <code>yyyy-MM-dd'T'HH:mm:ssXXX</code> or as an int representing seconds since the epoch (like
+     * <a href="https://reproducible-builds.org/docs/source-date-epoch/">SOURCE_DATE_EPOCH</a>).
+     *
+     * @since 1.9
+     */
+    @Parameter(defaultValue = "${project.build.outputTimestamp}")
+    private String outputTimestamp;
```

Comment:
If we parse `project.build.outputTimestamp` and `SOURCE_DATE_EPOCH` I don't think it's necessary to keep a mojo property, nobody is going to configure jdeb specifically if it's possible to do it at a higher level.

Reply:
Sorry I don't follow. I thought this is the way how to read the properties from pom.xml. Is there another way how to get the specified value?

Reply:
If you add the project to the mojo with:

    @Component
    private MavenProject project;

you can then call `project.getProperties()` to get the properties defined in the `pom.xml` file:

    project.getProperties().get("project.build.outputTimestamp");

Reply:
see https://maven.apache.org/guides/mini/guide-reproducible-builds.html

the property is defined globally, then available globally, and it is read by `maven-archiver` in `archiver.configureReproducible( outputTimestamp );`

see for example how maven-jar plugin has integrated the feature: https://github.com/apache/maven-jar-plugin/commit/64c5e6530b4712cd95501fffb2de6bb1a202cd89/

Reply:
Since this follows the example given above, I'd prefer to leave it as is.